news/2017-08-25-scorewatch-august-week-3.md

```diff
@@ -0,0 +1,57 @@
+
+[kablaze](https://osu.ppy.sh/u/3043603) takes the spotlight for this article as he stomps [dj TAKA - quaver](https://osu.ppy.sh/b/915210) with an earth-shattering HDHR 99.77% 2x miss score to earn his first 700pp score (exactly rounded up)! To think that he only got 1x 100 after all this blows our minds. This is incredible.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/SKvA60fooqM" frameborder="0" allowfullscreen></iframe></br>
+
+A score that was almost replicated 4 years back! [Rafis](https://osu.ppy.sh/u/2558286) steals the 2nd spot on [xi - FREEDOM DiVE](https://osu.ppy.sh/b/129891) with an unbelievable 99.93% FC to take 538pp! Cue flashbacks to Cookiezi’s 99.97% FC play back in 2013!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/7TTxN6ul7ko" frameborder="0" allowfullscreen></iframe></br>
+
+[Rafis](https://osu.ppy.sh/u/2558286) continues his dominance on this article as he sets the number one score on [xi - Ascension to Heaven](https://osu.ppy.sh/b/807074) with an unfortunate 98.98% 2555x 1 miss play to snag 637pp; an FC could have been worth more than 660pp! Will this map finally be conquered soon? Maybe it might, but in the meantime, you will have to check out this insane score! 
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/zByevoMS2ck" frameborder="0" allowfullscreen></iframe></br>
+
+[Motion](https://osu.ppy.sh/u/3885626) has taken the number one spot in osu!catch for the first time! To celebrate this, here is the man himself setting the an impressive 99.19% NC FC on [Hitsuji to Ojisan - XENO](https://osu.ppy.sh/b/663001) to claim 1004pp! Congratulations!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/wjplwLNaIMQ" frameborder="0" allowfullscreen></iframe></br>
+
+[EtienneXC](https://osu.ppy.sh/u/5610085) tears up [Evening’s](https://osu.ppy.sh/u/2193881) [Imperial Circus Dead Decadence - FUBUKI](https://osu.ppy.sh/b/1080145) by setting an amazing 99.65% FC. Standing at 4:14 mins of drain time, this play tests a player’s control and stamina limits. With an accuracy like this, it’s not doubt Etienne is one of the craziest 4K players out there today!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/uKDiGVBDlHY" frameborder="0" allowfullscreen></iframe></br>
+
+BLAST FROM THE PAST
+-----
+
+We blast ourselves back to 2015 where [rustbell](https://osu.ppy.sh/u/227717), known for his technical ability in maps, took an amazing 99.57% HDFL FC on [IOSYS - Kanbu de Todomatte Sugu Tokeru ~ Kyouki no Udongein](https://osu.ppy.sh/b/16266) to earn 211pp. Honestly speaking, you have to see it to believe it.
+
+Once again, do support osu!archive[https://www.reddit.com/r/osugame/comments/6uracb/osuarchive_4_rustbell_iosys_kanbu_de_todomatte/] (it is still ongoing on the osugame reddit) as they approach a week of amazing old scores!
```

Comment:
markdown bork here, is osu!archive meant to be linked? need to reverse the contents between the two brackets and ensure no whitespaces either